### PR TITLE
[RFR] Improve developer experience a bit

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: installer/js/github/index.js [hunk 1]
```diff
@@ -110,35 +110,32 @@ const getHookId = (accessToken, user, repository) => {
...
         })
-        .then(repositories => {
-            return Promise.all(
-                repositories.map(r => isSedyInstalled(accessToken, user, r))
-            ).then(sedyInstallations => repositories.map((repository, index) => ({
-                ...repository,
-                sedy_installed: sedyInstallations[index],
-            })));
+        .then((repositories) => {
+            return Promise.all(repositories.map(r => isSedyInstalled(accessToken, user, r)))
```
Comment (djhi) [inline installer/js/github/index.js]: loose the return

File: installer/package.json [hunk 2]
```diff
@@ -22,13 +22,14 @@
     "babel-preset-stage-1": "6.16.0",
     "config": "1.21.0",
     "css-loader": "0.25.0",
-    "file-loader": "0.9.0",
+    "extract-text-webpack-plugin": "^2.0.0-rc.3",
```
Comment (djhi) [inline installer/package.json]: avoid `^`

File: installer/package.json [hunk 3]
```diff
@@ -22,13 +22,14 @@
...
+    "file-loader": "^0.10.0",
```
Comment (djhi) [inline installer/package.json]: same

File: installer/package.json [hunk 4]
```diff
@@ -22,13 +22,14 @@
...
     "html-webpack-plugin": "2.24.0",
     "material-ui": "0.16.1",
     "react": "15.3.2",
     "react-dom": "15.3.2",
     "react-tap-event-plugin": "1.0.0",
-    "webpack": "1.13.3",
-    "webpack-dev-server": "1.16.2"
+    "webpack": "^2.2.1",
```
Comment (djhi) [inline installer/package.json]: same

File: installer/package.json [hunk 5]
```diff
@@ -22,13 +22,14 @@
...
+    "webpack-dev-server": "^2.3.0"
```
Comment (djhi) [inline installer/package.json]: same

File: oauth/package.json [hunk 6]
```diff
@@ -28,6 +28,6 @@
     "json-loader": "^0.5.4",
     "mocha": "^2.4.5",
     "request": "2.76.0",
-    "webpack": "1.13.3"
+    "webpack": "^2.2.1"
```
Comment (djhi) [inline oauth/package.json]: same in whole file

File: sedy/package.json [hunk 7]
```diff
@@ -33,6 +33,6 @@
     "mocha": "^2.4.5",
     "request": "2.76.0",
     "sinon": "^1.17.3",
-    "webpack": "1.13.3"
+    "webpack": "^2.2.1"
```
Comment (djhi) [inline sedy/package.json]: same in while file

**Comment by Kmaschta** [comment]
@djhi Reviewed & green

**Pull request merged**